packages/compass-connections-navigation/src/tree-item.tsx

```diff
@@ -23,7 +21,7 @@ const expandButton = css({
   // Not using leafygreen spacing here because none of them allow to align the
   // button with the search bar content. This probably can go away when we are
   // rebuilding the search also
-  padding: 7,
+  padding: 6,
```

Comment:
This doesn't match the design (along with other styles here), but probably worth opening another ticket to align those because there is a bunch of small inconsistencies in the list when compared to what's in figma:

This padding should be 8:

![image](https://github.com/mongodb-js/compass/assets/5036933/f937b48a-5fd7-430f-aefd-11028bfc9c56)

The icon next to it should be 14, not 16 px:

![image](https://github.com/mongodb-js/compass/assets/5036933/418f0594-4037-4eec-bcb2-930de4fb3d85)

Padding between icons and text is 8:

![image](https://github.com/mongodb-js/compass/assets/5036933/cc47e4c0-6d74-47a3-94e5-c0285ac3f64c)


There is a radius on a selected item background:

![image](https://github.com/mongodb-js/compass/assets/5036933/b8c9952d-c894-4c10-9d54-fddc05c6e55f)

We should also probably just use the ticket we will open to make the same styles apply in "single connection mode" so that we don't need to continuosly fix style issues in two places for no reason